lib/ansible/modules/cloud/azure/azure_rm_postgresqlserver_facts.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_postgresqlserver_facts
+version_added: "2.7"
+short_description: Get PostgreSQL Server facts.
+description:
+    - Get facts of PostgreSQL Server.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
```

Comment:
Let's just standardize on `resource_group` and `name` for these (feel free to leave `server_name` as an alias though); the resource type should already be clear from the module name in a playbook context